modules/avatars/lib/api/v3/users/user_avatar_api.rb

```diff
@@ -35,6 +35,11 @@ class UserAvatarAPI < ::API::OpenProjectAPI
         helpers ::API::Helpers::AttachmentRenderer
 
         get '/avatar' do
+          # Cache for one day
+          expire_in = 60 * 60 * 24
+          header "Cache-Control", "public, max-age=#{expire_in}"
+          header "Expires", CGI.rfc1123_date(Time.now.utc + expire_in)
```

Comment:
I'd exclude the current user, so that changes the user makes to the his own avatar are displayed right away.

Reply:
Good point.

Reply:
done!